test/OpenTelemetry.Exporter.Geneva.Tests/OtlpProtobufMetricExporterTests.cs

```diff
@@ -0,0 +1,237 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using System.Linq;
+using Google.Protobuf;
+using Google.Protobuf.Collections;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using Xunit;
+using OtlpCollector = OpenTelemetry.Proto.Collector.Metrics.V1;
+using OtlpCommon = OpenTelemetry.Proto.Common.V1;
+
+namespace OpenTelemetry.Exporter.Geneva.Tests;
+
+public class OtlpProtobufMetricExporterTests
+{
+    [Fact]
+    public void LongCounterSerializationSingleMetricPoint()
+    {
+        using var meter = new Meter(nameof(this.LongCounterSerializationSingleMetricPoint), "0.0.1");
+        var longCounter = meter.CreateCounter<long>("longCounter");
+        var exportedItems = new List<Metric>();
+        using var inMemoryReader = new BaseExportingMetricReader(new InMemoryExporter<Metric>(exportedItems))
+        {
+            TemporalityPreference = MetricReaderTemporalityPreference.Delta,
+        };
+
+        using var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .AddMeter(nameof(this.LongCounterSerializationSingleMetricPoint))
+            .AddReader(inMemoryReader)
+            .Build();
+
+        longCounter.Add(123, new("tag1", "value1"), new("tag2", "value2"));
+
+        meterProvider.ForceFlush();
+
+        var buffer = new byte[65360];
+
+        var testTransport = new TestTransport();
+        var otlpProtobufSerializer = new OtlpProtobufSerializer(testTransport);
+
+        otlpProtobufSerializer.SerializeAndSendMetrics(buffer, Resource.Empty, new Batch<Metric>(exportedItems.ToArray(), exportedItems.Count));
+
+        Assert.Single(testTransport.ExportedItems);
+
+        var request = new OtlpCollector.ExportMetricsServiceRequest();
+
+        request.MergeFrom(testTransport.ExportedItems[0]);
+
+        Assert.Single(request.ResourceMetrics);
```

Comment:
Add some resource attributes and test if the resources provided are contained in the serialized data.